src/index.test.ts

```diff
@@ -1,11 +1,13 @@
-import { Duplex } from 'stream';
 import { JsonRpcEngine } from '@metamask/json-rpc-engine';
 import PortStream from 'extension-port-stream';
+/* eslint import/no-nodejs-modules: ["error", {"allow": ["stream"]}] */
```

Comment:
Nit: It might be better to disable this rule entirely for `*.test.ts` files, since using Node.js packages is OK in tests. That rule is intended to help keep the package portable across Node.js and web, which doesn't apply to tests of course.

Reply:
The changes have been pushed.